util/stats/query_stats.go

```diff
@@ -25,6 +25,7 @@ const (
 	InnerEvalTime
 	ResultAppendTime
 	ExecQueueTime
+	ExecTotalTime
```

Comment:
Hm, we have now both `TotalEvalTime` and `ExecTotalTime`, which doesn't seem to follow a consistent naming pattern.

Reply:
Yep, can you make it `EvalTotalTime`?